src/main/kotlin/uk/gov/dluhc/printapi/config/CorrelationIdMdcConfiguration.kt

```diff
@@ -1,18 +1,26 @@
 package uk.gov.dluhc.printapi.config
 
+import mu.KotlinLogging
 import org.aspectj.lang.ProceedingJoinPoint
 import org.aspectj.lang.annotation.Around
 import org.aspectj.lang.annotation.Aspect
 import org.slf4j.MDC
+import org.springframework.http.HttpHeaders
 import org.springframework.messaging.Message
 import org.springframework.messaging.support.GenericMessage
 import org.springframework.stereotype.Component
+import org.springframework.web.reactive.function.client.ClientRequest
+import org.springframework.web.reactive.function.client.ClientResponse
+import org.springframework.web.reactive.function.client.ExchangeFilterFunction
+import org.springframework.web.reactive.function.client.ExchangeFunction
 import org.springframework.web.servlet.HandlerInterceptor
-import java.lang.Exception
+import reactor.core.publisher.Mono
 import java.util.UUID
 import javax.servlet.http.HttpServletRequest
 import javax.servlet.http.HttpServletResponse
 
+private val logger = KotlinLogging.logger {}
```

Comment:
Is this used?
Seems a bit odd to be logging stuff in beans and components that are all about adding stuff to the MDC for logging ....

Reply:
no it's not, need to remove.
I originally had the idea of logging a warning in cases where the inbound request does not have a correlation-id and we have to generate a new one. This is quite useful when viewing logs and you can easily spot outliers who haven't sent a correlation-id. I didn't end up doing this since it would require a bit of a refactor.